src/test/units/PipelineStore.test.js

```diff
@@ -0,0 +1,25 @@
+import {expect} from "chai";
+import PipelineStore from "../../js/plugin/sdk/pipeline/PipelineStore";
+import pipelineModel from "../../js/plugin/sdk/pipeline/PipelineModel";
+import PipelineOperations from "../../js/plugin/sdk/pipeline/PipelineOperations";
+
+describe("PipelineStore tests", function () {
+
+    beforeEach(function (){
+      PipelineStore.pipeline.clearPipeline();
+    });
+
+    it ("should share the same filters between multiple pipeline instances", function (){
+      const filter = function (data) {return data;};
+      const filter2 = function (data) {return data;};
+
+      const pipeline_one = PipelineStore;
```

Comment:
Done. Don't know if this suite makes sense at all, I can't import `PipelineStore` twice so I can't even simulate two instance of `Pipelinestore`. What do you think about removing this suite completely?

About the forwarding test, I would just register two operators and check that they ran, this would confirm that `PipelineStore` works as expected. What do you think?

Reply:
You should be able to import the store as follows:

```js
import {
  default as TestPipelineStore
} from "../../js/plugin/sdk/pipeline/PipelineStore";

// ...

PipelineStore.pipelines === TestPipelineStore.pipelines;
```

I'd use `sinon` to check wether the respective functions were called as illustrated in the following:

```js
sinon.spy(PipelineStore.pipeline, "applyPipeline");

expect(PipelineStore.pipeline.applyPipeline).to.be.true;
```